airbyte-cdk/python/airbyte_cdk/sources/utils/stream_helpers.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from typing import Any, Mapping, Optional
+
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.core import StreamData
+
+
+class StreamHelper:
+    def get_first_record(self, stream: Stream) -> StreamData:
+        """
+        Gets the first record for a stream.
+
+        :param stream: stream
+        :return: StreamData containing the first record in the stream
+        """
+        # Some streams need a stream slice to read records (e.g. if they have a SubstreamSlicer)
+        stream_slice = self.get_stream_slice(stream)
+        records = stream.read_records(sync_mode=SyncMode.full_refresh, stream_slice=stream_slice)
+        next(records)
```

Comment:
`return next(records)`? This method will always return `None` right now

Reply:
This whole file isn't supposed to be here - It's not being used anywhere, made it into accidentally from the merge. Thanks for flagging! The real utils file is the one we added last week (hence not in the diff) imported [here](https://github.com/airbytehq/airbyte/pull/21484/files/91affb8c69d7c532438ac4f3f2ffc1f6387fbb33#diff-a9dd5cf1051977e9a7c58329dbac9bded08d9f53f314ae2bdeef073f360820bcR12). Removing this one